docs/metrics.asciidoc

```diff
@@ -0,0 +1,102 @@
+ifdef::env-github[]
```

Comment:
This file needs to be included in `index.asciidoc` for it to build:
```js
include::./metrics.asciidoc[Metrics]
```

Reply:
Thanks, added - trying to have the same order as other agents, let me know if I should move it.

Reply:
That's good for now. Thanks!